specification/securitydevops/resource-manager/Microsoft.SecurityDevOps/preview/2022-09-01-preview/securitydevops.json

```diff
@@ -0,0 +1,2692 @@
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "repoId": {
+          "description": "Gets or sets Azure DevOps repo id.",
+          "type": "string"
+        },
+        "repoUrl": {
+          "description": "Gets or sets AzureDevOps repo url.",
+          "type": "string"
+        },
+        "orgName": {
+          "description": "Gets or sets AzureDevOps Org Name.",
+          "type": "string"
+        },
+        "projectName": {
+          "description": "Gets or sets AzureDevOps Project Name.",
+          "type": "string"
+        },
+        "visibility": {
+          "description": "Gets or sets AzureDevOps repo visibility, whether it is public or private etc.",
+          "type": "string"
+        },
+        "actionableRemediation": {
+          "$ref": "#/definitions/ActionableRemediation"
+        }
+      }
+    },
+    "GitHubConnector": {
+      "description": "Represents an ARM resource for /subscriptions/xxx/resourceGroups/xxx/providers/Microsoft.SecurityDevOps/gitHubConnectors.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/GitHubConnectorProperties"
+        }
+      }
+    },
+    "GitHubConnectorListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets list of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GitHubConnector"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "Gets or sets next link to scroll over the results.",
+          "type": "string"
+        }
+      }
+    },
+    "GitHubConnectorProperties": {
+      "description": "Properties of the ARM resource for /subscriptions/xxx/resourceGroups/xxx/providers/Microsoft.SecurityDevOps/gitHubConnectors.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "code": {
+          "description": "Gets or sets one-time OAuth code to exchange for refresh and access tokens.",
+          "type": "string",
+          "example": "00000000000000000000.",
+          "x-ms-secret": true
+        }
+      }
+    },
+    "GitHubConnectorStats": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/GitHubConnectorStatsProperties"
+        }
+      }
+    },
+    "GitHubConnectorStatsListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets list of resources.",
```

Comment:
>resources [](http://example.com/codeflow?start=47&length=9)

Be specific about the kind of resource this list is, since its a particular specialized list type.

Reply:
hopefully we will deprecate this API from next API version as resources will be onboarded to Azure Resource Grouph. This is a pseudo resource to support counts on the proxy resources.